src/Compilers/CSharp/Test/Syntax/Syntax/LambdaUtilitiesTests.cs

```diff
@@ -43,8 +43,10 @@ void M()
             bool expected = enclosingMethod.MethodKind == MethodKind.LambdaMethod && enclosingSyntax.Span.Contains(span.Value);
 
             var node = tree.GetRoot().FindNode(span.Value);
-            Assert.Equal(expected, LambdaUtilities.IsLambdaBody(node));
-            Assert.Equal(isLambdaBody, expected);
+            Assert.False(isLambdaBody && isReducedLambdaBody);
+            Assert.Equal(expected, LambdaUtilities.IsLambdaBody(node, allowReducedLambdas: true));
+            Assert.Equal(isLambdaBody || isReducedLambdaBody, expected);
```

Comment:
>Assert.Equal(isLambdaBody || isReducedLambdaBody, expected); [](start = 12, length = 60)

Consider using ```expected``` as the first argument, otherwise failure message is going to be misleading.